src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1560,35 +1560,26 @@
      * </pre>
```

Comment:
could you please fix example while you are on this token type, please put space after `if`